src/components/Board/BoardShare/BoardShare.component.js

```diff
@@ -26,6 +27,12 @@ import messages from './BoardShare.messages';
 
 import './BoardShare.css';
 
+function OpenLogWd(){
```

Comment:
function names should always start with lower case. See coding convention: https://www.robinwieruch.de/javascript-naming-conventions